test/unit/DCAHub/dca-hub-position-handler.spec.ts

```diff
@@ -1082,56 +1083,6 @@ contract('DCAPositionHandler', () => {
         expect(swapped).to.equal(0);
       });
     });
-
-    describe('verify overflow errors', () => {
```

Comment:
I deleted overflow tests, because as we explained before, we will never reach a scenario like this. Also, with the new division, the overflow is more difficult to generate